<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Market-Analysis.qrc">
    <normaloff>:/img/diagram_64.png</normaloff>:/img/diagram_64.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="vLayoutCentral">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QTabWidget" name="vTabWidget">
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Kit"/>
    <addaction name="actionOpen_Kit"/>
    <addaction name="actionSave_Kit"/>
    <addaction name="actionClose_Kit"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionKit_Configuration"/>
    <addaction name="actionTrain_NN"/>
    <addaction name="actionStart_forecasting"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionDelete_Kit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Kit"/>
   <addaction name="actionOpen_Kit"/>
   <addaction name="separator"/>
   <addaction name="actionTrain_NN"/>
   <addaction name="actionStart_forecasting"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionKit_Configuration"/>
   <addaction name="actionDelete_Kit"/>
  </widget>
  <action name="actionNew_Kit">
   <property name="icon">
    <iconset resource="Market-Analysis.qrc">
     <normaloff>:/img/document_64.png</normaloff>:/img/document_64.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New Kit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="Market-Analysis.qrc">
     <normaloff>:/img/gear_64.png</normaloff>:/img/gear_64.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionSave_Kit">
   <property name="icon">
    <iconset resource="Market-Analysis.qrc">
     <normaloff>:/img/save_64.png</normaloff>:/img/save_64.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save Kit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDelete_Kit">
   <property name="icon">
    <iconset resource="Market-Analysis.qrc">
     <normaloff>:/img/warning_64.png</normaloff>:/img/warning_64.png</iconset>
   </property>
   <property name="text">
    <string>Delete Kit</string>
   </property>
  </action>
  <action name="actionOpen_Kit">
   <property name="icon">
    <iconset resource="Market-Analysis.qrc">
     <normaloff>:/img/folder_64.png</normaloff>:/img/folder_64.png</iconset>
   </property>
   <property name="text">
    <string>Open Kit</string>
   </property>
  </action>
  <action name="actionTrain_NN">
   <property name="icon">
    <iconset resource="Market-Analysis.qrc">
     <normaloff>:/img/clipboard_pencil_64.png</normaloff>:/img/clipboard_pencil_64.png</iconset>
   </property>
   <property name="text">
    <string>Train NN</string>
   </property>
   <property name="toolTip">
    <string>Train Neural Network</string>
   </property>
  </action>
  <action name="actionStart_forecasting">
   <property name="icon">
    <iconset resource="Market-Analysis.qrc">
     <normaloff>:/img/statistics2_diagram_64.png</normaloff>:/img/statistics2_diagram_64.png</iconset>
   </property>
   <property name="text">
    <string>Start forecasting</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="Market-Analysis.qrc">
     <normaloff>:/img/stop_64.png</normaloff>:/img/stop_64.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop work</string>
   </property>
  </action>
  <action name="actionKit_Configuration">
   <property name="icon">
    <iconset resource="Market-Analysis.qrc">
     <normaloff>:/img/gear_64.png</normaloff>:/img/gear_64.png</iconset>
   </property>
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="Market-Analysis.qrc">
     <normaloff>:/img/help_64.png</normaloff>:/img/help_64.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="Market-Analysis.qrc">
     <normaloff>:/img/info_64.png</normaloff>:/img/info_64.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionClose_Kit">
   <property name="icon">
    <iconset resource="Market-Analysis.qrc">
     <normaloff>:/img/delete_64.png</normaloff>:/img/delete_64.png</iconset>
   </property>
   <property name="text">
    <string>Close Kit</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
   <property name="toolTip">
    <string>About Qt Framework</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Market-Analysis.qrc"/>
 </resources>
 <connections/>
</ui>
